roofit/roofitcore/src/RooAbsData.cxx

```diff
@@ -26,15 +26,11 @@ points for its contents and provides an iterator over its elements
 **/
 
 #include "RooAbsData.h"
-#include "RooFit.h"
-
-#include <iostream>
 
 #include "TBuffer.h"
 #include "TClass.h"
 #include "TMath.h"
 #include "TTree.h"
-#include "strlcpy.h"
```

Comment:
I think you need to keep this: `strlcpy` is used twice in this file, and without the include it doesn't compile for me

Reply:
I fixed it by going to RooHelpers::tokenise to tokenise the strings.